org.openhab.binding.zigbee/src/main/java/org/openhab/binding/zigbee/handler/ZigBeeCoordinatorHandler.java

```diff
@@ -439,6 +445,13 @@ private synchronized void initialiseZigBee() {
                 return;
         }
 
+        // Add all the clusters that we are supporting.
+        // If we don't do this, the framework will reject any packets for clusters we have not stated support for.
+        channelFactory.getImplementedClientClusters().stream()
+                .forEach(clusterId -> networkManager.addSupportedClientCluster(clusterId));
+        channelFactory.getImplementedServerClusters().stream()
```

Comment:
Could be reduced to `channelFactory.getImplementedServerClusters().forEach(networkManager::addSupportedServerCluster)`